src/redisearch_rs/rqe_iterators_bencher/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "rqe_iterators_bencher"
+version.workspace = true
+edition.workspace = true
+license-file.workspace = true
+publish.workspace = true
+
+[lints]
+workspace = true
+
+[lib]
+# See https://bheisler.github.io/criterion.rs/book/faq.html#cargo-bench-gives-unrecognized-option-errors-for-valid-command-line-options
+bench = false
+
+[[bench]]
+name = "iterators"
+harness = false
+
+[build-dependencies]
+bindgen.workspace = true
+build_utils = { path = "../build_utils" }
+
+[dependencies]
+criterion.workspace = true
+ffi.workspace = true
+inverted_index.workspace = true
+itertools.workspace = true
+redis_mock.workspace = true
+rqe_iterators.workspace = true
+types_ffi = { path = "../c_entrypoint/types_ffi" }
+
+[target.'cfg(all(target_env="musl", target_os="linux"))'.dependencies.redis-module]
+# Statically link to the libclang on aarch64-unknown-linux-musl,
+# necessary on Alpine.
+# See https://github.com/rust-lang/rust-bindgen/issues/2360
+features = ["bindgen-static", "min-redis-compatibility-version-6-0"]
+workspace = true
+default-features = false
+
+[target.'cfg(not(all(target_env="musl", target_os="linux")))'.dependencies.redis-module]
+workspace = true
+default-features = true
```

Comment:
Can this blurb be moved into the workspace Cargo.toml, so we can just `.workspace = true` in each crate rather than having to copy-paste it?